subservers/manager.go

```diff
@@ -78,6 +78,20 @@ func (s *Manager) AddServer(ss SubServer, enable bool) {
 	s.statusServer.SetEnabled(ss.Name())
 }
 
+// GetServer returns the sub-server with the given name if it exists.
+func (s *Manager) GetServer(name string) (SubServer, bool) {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+
+	for _, ss := range s.servers {
+		if ss.Name() == name {
```

Comment:
adding [this PR](https://github.com/lightninglabs/lightning-terminal/pull/877) which changes the storage of subservers to be a map indexed by name. Then we dont need to iterate here.

Reply:
Okay, will rebase this PR once the other one is in and then update this.

Reply:
Updated.